### ISSUE TITLE
Make it easier to add fake users with specific playtime for testing purposes

### ISSUE THREAD

**Issue by jauggy** [beyond-all-reason/teiserver#251]
Make it easier to add fake users with specific playtime or adjust existing users to have specific player time. Player time is used to calculate rank on login. This feature would be useful for testing.

**Comment by jauggy** [comment]
To bypass email checks we can change this setting:
![Screenshot 2024-04-14 210752](https://github.com/beyond-all-reason/teiserver/assets/1305569/3df5773f-cd16-40a3-8abc-1224de6a0cd3)

I still want a way to set play time though to test various chev levels.

**Comment by L-e-x-o-n** [comment]
1. Run server in interactive mode with `iex -S mix phx.server`
2. Go to http://localhost:4000/teiserver/admin/site#Profiles and set **Rank method** to `role`
3. Find out the user ID of the user you want to test (http://localhost:4000/profile/<ID> for example)
4. Execute command in iex: `Teiserver.Account.update_user_stat(<ID>, %{"player_minutes"=> 100*60, "spectator_minutes"=>20*60, "total_minutes"=>130*60})` (you can only set `spectator_minutes` and `player_minutes`, only those are used for rank calculation)
5. Login as the user in game (playtime will be updated after the previous step, but rank is only recalculated on login)
6. Confirm that your rank changed (for example by visiting http://localhost:4000/profile/<ID>/playtime)

**Comment by jauggy** [comment]
So should we put a guide for this and link it somewhere?

**Comment by L-e-x-o-n** [comment]
Not necessarily, I am just showing an example of how it can be done if you need it right now for some tests. The issue is still valid and an easier way to do it would be nice.
